ldj-spring-renderer-example/src/main/java/com/infosupport/ldoc/springexample/ampq/QueueInteractionKind.java

```diff
@@ -0,0 +1,6 @@
+package com.infosupport.ldoc.springexample.ampq;
+
+public enum QueueInteractionKind {
+  READ,
+  POST
```

Comment:
Send and receive are indeed used by the spring docs. However, we also had the option to merge the "event" and "queue" interactions. If that happens, it would not make sense to change these names now.
Furthermore, spring also uses the names publisher and consumer in the context of queues as roles. So that would, in my opinion, also be an option.